app/assets/v2/js/wallet.js

```diff
@@ -122,6 +122,7 @@ async function fetchAccountData(provider) {
   // MetaMask does not give you all accounts, only the selected account
   console.log('Got accounts', accounts);
   selectedAccount = accounts[0] || provider.account;
+  // selectedAccount = '0x1402e01ba957e030200ec21e1417dc32c11e486b';
```

Comment:
Should this be removed before we push it out?

Reply:
i kept just for the sake of testing ! will remove it out once we test it out in live as well